magento/magento2
# Fixed the welcome message contains single quote character

### Description (*)

This PR is fixed for the issue, that if we put the single quote character to the header welcome message config, it will lead the frontend to have a bunch of console errors:

> Uncaught SyntaxError: Unable to process binding "ifnot: function(){return customer().fullname }"
Message: Unable to parse bindings.


### Manual testing scenarios (*)

1. From Admin, navigate to Content -> Design -> Configuration
2. Select a theme and Edit
3. In the edit page, expand the Header setting, put the `Welcome Text`, like "Welcome Khoi's store!"
4. Clear cache, then go to frontend, you'll see a JS error, and some JS functions not working because of this issue.

### Questions or comments
Maybe we need to bind the message as a text, instead of HTML binding, like:

> data-bind="text: '<?= $escaper->escapeHtmlAttr($welcomeMessage) ?>'"

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#35481: Fixed the welcome message contains single quote character
